src/bin/zmcontactbackup

```diff
@@ -0,0 +1,565 @@
+#!/usr/bin/perl
+#
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2018 Synacor, Inc.
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+
+=head1 NAME
+
+zmcontactbackup - start/stop/schedule contact backup on mailbox server(s)
+
+=head1 SYNOPSIS
+
+zmcontactbackup <-h|-m|-r|-p|-q|-d|-f|[-a backup_schedule]> [mailbox_server...]
+
+       -r|--start          : start contact backup
+       -p|--stop           : stop contact backup
+       -h|--help           : print a brief help message
+       -m|--man            : print full man message
+       -q|--query          : prints existing schedule
+       -a|--append         : append contact backup schedule
+       -f|--flush          : flush all contact backup schedules
+       -d|--default        : flush all schedules and set default schedule to srart (default = 3 am everyday on all mailbox servers)
```

Comment:
typo 'srart' should be 'start'?
perhaps change/simplify the description to say something like:
```(re)set to the default (3am daily) backup schedule ```

Reply:
done